docs/reference/commandline/create.md

```diff
@@ -55,7 +55,12 @@ Creates a new container.
       --memory-swap=""              Total memory (memory + swap), '-1' to disable swap
       --memory-swappiness=""        Tune a container's memory swappiness behavior. Accepts an integer between 0 and 100.
       --name=""                     Assign a name to the container
-      --net="default"               Set the Network mode for the container
+      --net="bridge"                Connects a container to a network
+                                    'bridge': creates a new network stack for the container on the docker bridge
+                                    'none': no networking for this container
+                                    'container:<name|id>': reuses another container network stack
+                                    'host': uses the host network stack inside the container
+                                    'NETWORK': connects the container to user-defined network using `docker network create` command, either the name or id can be used.
```

Comment:
@wenchma Thank you for the contribution.  I think there are few things in this that need to be tighten.  For example, `create new` is redundant as create implies something didn't exist before.  Omit phrases that refer back to the container --- the intro sentence makes the unnecessary. Also, we should resist the temptation to provide too much information in the option --- for the `NETWORK` option the description is sliding into user info.  We should rely on the description of the examples  for that --- there are some in Examples for connecting to a network.  

This brings it closer:

```markdown
--net="bridge"         Connect a container to a network
                              'bridge': create a network stack on the default Docker bridge
                              'none': no networking
                              'container:<name|id>': reuse another container's network stack
                              'host': use the Docker host network stack
                              'NETWORK': connect to a user-defined network
```